rpc/block_test.go

```diff
@@ -702,3 +702,119 @@ func TestRpcBlockAdaptation(t *testing.T) {
 		require.Equal(t, &felt.Zero, blockWithTxHashes.BlockHeader.SequencerAddress)
 	})
 }
+
+func TestJunoGetBlockWithTxsAndReceipts(t *testing.T) {
+	blockId := map[string]rpc.BlockID{
+		"latest":  {Latest: true},
+		"pending": {Pending: true},
+		"hash":    {Hash: new(felt.Felt).SetUint64(1)},
+		"number":  {Number: 1},
+	}
+
+	for desc, id := range blockId {
+		t.Run(desc, func(t *testing.T) {
+			log := utils.NewNopZapLogger()
+			n := utils.Ptr(utils.Mainnet)
+			chain := blockchain.New(pebble.NewMemTest(t), n)
+			handler := rpc.New(chain, nil, nil, "", n, log)
```

Comment:
nit : I think we can push this outside of the loop

Reply:
resolved